types/resize-observer-browser/index.d.ts

```diff
@@ -9,16 +9,33 @@ interface Window {
     ResizeObserver: typeof ResizeObserver;
 }
 
+interface ResizeObserverObserveOptions {
+    /**
+     * Sets which box model the observer will observe changes to. Possible values
+     * are `content-box` (the default), and `border-box`.
+     *
+     * @default "content-box"
+     */
+    box?: 'content-box' | 'border-box';
+}
+
+interface ResizeObserverEntryBoxSize {
+    blockSize: number;
+    inlineSize: number;
+}
+
 declare class ResizeObserver {
     constructor(callback: ResizeObserverCallback);
     disconnect(): void;
-    observe(target: Element): void;
+    observe: (target: Element, options?: ResizeObserverObserveOptions) => void;
```

Comment:
Rename the type of options to `ResizeObserverOptions`, as this follows the spec at https://drafts.csswg.org/resize-observer/#dictdef-resizeobserveroptions.

Reply:
Also, why the different style of specifying the function type for the observe member vs. using the member shorthand?

Reply:
Good catch. This shouldn't be using the syntax you changed to.

should read:

`observe(target: Type, options?: Type): void;`

Reply:
Good catch! Applying these changes now